core/src/main/java/org/apache/brooklyn/enricher/stock/aggregator/AggregationJob.java

```diff
@@ -32,6 +33,8 @@
 import org.apache.brooklyn.core.sensor.BasicAttributeSensorAndConfigKey;
 
 @Beta
+@Deprecated
+/** @Deprecated since 1.0.0 No planed replacement */
```

Comment:
1.0.0 was already released, this message should not be since 1.1.0?

Reply:
You are absolutely right @jcabrerizo, I'll update that